src/api/repositories/PetRepository.ts

```diff
@@ -8,23 +8,30 @@ const pet_entity_database = [
 ];
 
 export class PetRepository {
-    public insert(): any {
-        console.log('Error, method not implemented');
+    public insert(pet: Pet): Pet {
+        pet.id = 4;
```

Comment:
- @Aristevao update here to auto-generate our id:
```typescript
pet.id = pet_entity_database.length;
```
- It's not the best way to this, but it's enough